plugins/scaffolder-backend/src/scaffolder/stages/helpers.test.ts

```diff
@@ -55,6 +55,7 @@ describe('Helpers', () => {
               },
             },
           },
+          owner: 'example@email.com',
```

Comment:
all examples of owners should be changed from emails to just example group names, so as not to give the impression that emails are typical / correct values :)